dev/integration_tests/abstract_method_smoke_test/lib/main.dart

```diff
@@ -48,9 +52,8 @@ class SecondPage extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
       body: Column(
-        children: <Widget>[
-          TextFormField(),
-          const Expanded(
+        children: const <Widget>[
+          Expanded(
```

Comment:
This seems like a problem with the embedding - tapping on a text field at the wrong moment shouldn't cause a WAKE_LOCK or a crash to fire, right?